### ISSUE TITLE
Keep navigation open

### ISSUE THREAD

**Issue by Brugman** [litespeedtech/cyberpanel#788]
Why does the main menu have to close when I click inside the page? It's confusing and adds no value.

When clicking links it's also useless: Let's say I click Users, and then List Users. At that point the menu will collapse, while the page is loading, and when the List Users page has loaded the menu will be open and List Users will be highlighted. But then when I click inside the page it will close again. This serves no purpose.

If I click Users and then Websites, then Users should close. That's good.

If I click Users and then Users, then Users should close. That's good.

![keep-subnav-open](https://user-images.githubusercontent.com/2203813/148863704-0c04d300-13bd-4412-acb4-119a534c8fe2.gif)

Please thumbs up if you agree.

**Comment by PowerChaos** [comment]
This is just javascript and when you click out of the page it auto collapse back to the original state
does not seems to me that it is a wrong thing , it prevent clustering of the page or to keep all tabs open that are not used
if you click on users , then on packages then the menu also collapse and it is the same trigger if you press outside the menu

Greetings from PowerChaos

**Comment by Brugman** [comment]
My explanation wasn't that good. Let me try again.

Let's say you visit the List Users page. You scroll down, see what's it about. Then you look at the menu, which is still open, to see what other admin pages there are that are related to "List Users".

If you never click on the page that works fine.

But if you are used to clicking on a blank part of a website to make sure it has focus before you scroll etc, then the entire Users subnav list has collapsed and you can't "see where you are" anymore.

If the menu would only open and collapse when you click it's, or anothers, top level link, that would avoid this annoyance, and it would not hinder anyone who is used to the current default.

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
